pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -172,7 +172,7 @@ public void setLocalTempDir(String localTempDir) {
   }
 
   public String getLocalTempDir() {
-    return getString(LOCAL_TEMP_DIR, getDataDir());
```

Comment:
Should we put any default value for local temp dir? It may not be set in the config though.

Reply:
Saw your code below. Can we add some comment on missing temp dir?

Reply:
There are comments in ControllerFilePathProvider (the only caller to this method)